app/utilities/schema.py

```diff
@@ -139,7 +148,7 @@ def _load_schema_file(schema_name, language_code):
             language_code=language_code,
         )
 
-    if not schema_exists(language_code, schema_name):
+    if not _schema_exists(language_code, schema_name):
```

Comment:
This is a duplicate of the call to _schema_exists on line 141; the result of this function could be set to a var and re-used

Reply:
Didn't catch it since it was existing but good shout, will fix.

Reply:
Actually looking at the code, it is done like that because it checks if the schema exists for the requested non default language code, then if it doesn't it sets language code to "en" and then checks that the "en" schema exists.